apps/ssr/.browserslistrc

```diff
@@ -9,9 +9,3 @@
 #   npx browserslist
 
 last 1 Chrome version
-last 1 Firefox version
```

Comment:
@edbzn is this fine for the ssr setup or do we require having those settings here?

Reply:
It's ok.